docker-compose.yaml

```diff
@@ -1,23 +1,4 @@
-version: '3'
-
 services:
-  scylla:
-    image: scylladb/scylla:latest
-    networks:
-      - scylla
-    volumes:
-      - ./data/scylla:/var/lib/scylla
-    ports:
-      - "8000:8000"
-    command: "--smp 2 --memory 2048M --alternator-port 8000 --alternator-write-isolation always_use_lwt"
-
-  cassandra:
-    image: cassandra:latest
-    networks:
-      - scylla
-    volumes:
-      - ./data/cassandra:/var/lib/cassandra
-
```

Comment:
I expect this is removed and left to user to start it, doesn't it break GH action tests?

Reply:
it does, https://github.com/scylladb/scylla-migrator/blob/master/.github/workflows/tests.yml depends on above (or am I missing something?)